config/prod.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  mongoUrl: process.env.mongoUrl
```

Comment:
why make a file for this ?

Reply:
We are going to use that to deploy in heroku